src/components/AnswerCard.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { FiArrowUpRight } from 'react-icons/fi'
+import styled from "styled-components";
+import graphql from 'babel-plugin-relay/macro'
+import { createFragmentContainer } from 'react-relay'
+
+import { Text } from '../styles'
+
+function AnswerCard(props) {
+  const { id,
+    answer,
+    text,
+    citationUrl,
+    citationTitle,
+    credibleCount,
+    notCredibleCount } = props.answer
+  const setColor = (answer === 'True') ? true : false;
+
+  return <div key={id}>
+    <>
+      <AnswerHeader children={answer} setColor={setColor} />
+    </>
+    <>
+      <Text.Small>Asked by <b>DEMO</b> </Text.Small>
+    </>
+    <>
+      <Text.Small children={text} />
+    </>
+    <MediaWrapper>
+      <FiArrowUpRight />
+      <MediaLink onClick={event => { event.stopPropagation(); }} href={citationUrl} >{citationUrl}</MediaLink>
+      <MediaTitle>{citationTitle}</MediaTitle>
+    </MediaWrapper>
+    <ButtonWrapper>
+      <VoteButton background={`background: #23BE7B;`}>
+        {credibleCount} Credible
+      </VoteButton>
+      <VoteButton background={`background: #E55934;`}>
+        {notCredibleCount} Not Credible
+      </VoteButton>
+    </ButtonWrapper>
+  </div>
+}
+
+const AnswerHeader = styled(Text.H2)`
+  margin-bottom: 1rem;
+  ${({ setColor }) => setColor ? `color: #23BE7B` : `color: #E55934`}
```

Comment:
These colors should be coming from a universal color style file. We don't want random hex colors floating around if we change our color scheme down the road.